tools/ci_build/provider_rewriter.py

```diff
@@ -0,0 +1,321 @@
+'''rewrite execution providers to disable ops'''
```

Comment:
nit: provider_ops_disabler.py or provider_ops_selector.py?

Reply:
Then let it be provider_ops_disabler.py.

Reply:
Done.